service-coordtransform/README.md

```diff
@@ -0,0 +1,3 @@
+### Paikkatietoikkuna.fi Coordinate Transform service
+
+Paikkatietoikkuna.fi Coordinate Transform for high precision transformations between different coordinate systems.
```

Comment:
Add some mention about the config like: 

Requires configuration for oskari-ext.properties: 

      coordtransform.endpoint=http://service.for/transform